experiments/react-native-uifabric/src/components/RadioGroup/RadioGroup.types.ts

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { ViewProps } from 'react-native';
+import { ITextProps } from '../Text';
+import { IViewWin32Props } from '@office-iss/react-native-win32';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { ITextTokens } from '../../tokens/TextTokens';
+
+export const radioGroupName = 'RadioGroup';
+
+export interface IRadioGroupState {
+  /*
+   ** The currently selected RadioButton's key
+   */
+  selectedKey: string;
+
+  /*
+   ** Updates the selected button and calls the client’s onChange callback
+   */
+  onChange: (key: string) => void;
+}
+
+export interface IRadioGroupProps {
+  /*
+   ** Descriptive label for the RadioGroup. This will be displayed as the title of the radio group to the user
+   */
+  label: string;
+
+  /*
+   ** The key of the RadioButton that will initially be selected
+   */
+  defaultSelectedKey: string;
```

Comment:
I think this prop should be optional. If it's not defined, then none of the radio button is selected.

Reply:
Changed to optional